ui/components/app/confirm/info/row/row.tsx

```diff
@@ -100,11 +101,24 @@ function RowAlert({
     return null;
   }
 
+  const [alertModalVisible, setAlertModalVisible] = useState<boolean>(false);
+  const handleCloseModal = () => {
+    setAlertModalVisible(false);
+  };
+
   return (
     <>
+      {alertModalVisible && (
```

Comment:
The more logic we get in here, I feel more comfortable with the `ConfirmInfoAlertRow` idea from the previous PR, to keep the two decoupled.